src/AppBundle/Entity/Post.php

```diff
@@ -95,7 +95,7 @@ class Post
      *      mappedBy="post",
      *      orphanRemoval=true
      * )
-     * @ORM\OrderBy({"publishedAt" = "DESC"})
+     * @ORM\OrderBy({"publishedAt": "DESC"})
```

Comment:
recent versions of doctrine/annotations allow both ``=`` and ``:`` as delimiter between the array key and value. Old versions were allowing only ``=``, which is why the ORM doc uses it.